ciao-controller/compute.go

```diff
@@ -906,7 +911,16 @@ func createServer(w http.ResponseWriter, r *http.Request, context *controller) {
 	}
 	servers.TotalServers = len(instances)
 
-	b, err := json.Marshal(servers)
+	// set machine ID for OpenStack compatibility
+	server.Server.ID = instances[0].ID
```

Comment:
If the code arrives to that line, it would mean at least one instance was created otherwise it would fall into 907 (before)

https://github.com/leoswaldo/ciao/blob/4aa603d12b1238600b0e3b6a27d70b864f13db32/ciao-controller/compute.go#L907